docs/3_finetuner_in_action/3_2_image_to_image.md

```diff
@@ -1,3 +1,117 @@
 # Image to image search using ResNet
 
-Coming soon!
\ No newline at end of file
+This guide will show you how to fine-tune a ResNet model for image-to-image retrieval.
```

Comment:
Perhaps we should standardize whether we want to hyphenate `image-to-image` or write instead `image to image` and make it consistent over all docs, including `text to text` etc.

Reply:
Good point. Let's stick with `image to image`, `text to text`, etc.